src/js/views/patients/schedule/filters_views.js

```diff
@@ -3,19 +3,15 @@ import { View } from 'marionette';
 
 import 'scss/modules/buttons.scss';
 
-import Droplist from 'js/components/droplist';
-
 import './schedule.scss';
 
 const FiltersView = View.extend({
```

Comment:
Is this view needed?

Reply:
Is this file needed even

Reply:
I don't think it is. In the last push I removed this `/schedule/filters_views.js` file and the `/worklist/filters_views.js` files. And moved the `AllFiltersButtonView` to the `schedule_views.js` and `worklist_views.js` files.